backend/supabase/user_contract_metrics/triggers.sql

```diff
@@ -0,0 +1,38 @@
+alter table user_contract_metrics
```

Comment:
Converting `user_contract_metrics` to native columns should totally be a single PR. It's possible that Ian or Marshall might have opinions, but it's buried within other changes here.